### PR TITLE
chore: add strict engine, and version for volta

### LINKED ISSUE #13: Strict node / npm version
### Is your feature request related to a problem? Please describe.
Restrict the project to LTS of node / npm

### Describe the solution you'd like.
Should also include restriction when using nvm

### PR DESCRIPTION
**Pull request by frontend-specialisten**
## Description
Add strict engine for node/NPM and pinned node version for volta

closes #13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,5 +96,8 @@
     "ts-jest": "27.0.3",
     "ts-node": "~9.1.1",
     "typescript": "~4.3.5"
+  },
+  "volta": {
+    "node": "14.17.5"
```
Comment (dzhavat) [inline package.json:101]: Suggestion: Add a comment similar to line 5 since this version should also be in sync.

File: package.json [hunk 2]
```diff
@@ -2,6 +2,12 @@
   "name": "energinet",
   "version": "0.0.0",
   "license": "MIT",
+  "//engines-comment": "Keep this in sync with .nvmrc",
```
Comment (dzhavat) [inline package.json:5]: Nice way to add comments :)

File: package.json (hunk 1 above)
Comment (frontend-specialisten) [reply package.json:101]: Good point!

**Comment by dzhavat** [comment]
Question: What version of npm did you use to generate the project. When I install with v6.14.14 (node v14.17.5) I get the following warning:

image

Can we fix that somehow as well?

**Review by dzhavat — APPROVED** [review]

**Pull request merged**